pkg/cli/create/scheme.go

```diff
@@ -0,0 +1,38 @@
+package create
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+
+	"github.com/openshift/api/quota"
+	"github.com/openshift/api/route"
+
+	schemehelper "github.com/openshift/oc/pkg/helpers/scheme"
+)
+
+var (
+	createCmdScheme = runtime.NewScheme()
+	createCmdCodecs = serializer.NewCodecFactory(createCmdScheme)
+)
+
+func init() {
+	schemehelper.InstallSchemes(createCmdScheme)
```

Comment:
Are we sure that we don't need to also install kube resources just you did in expose?

Reply:
I will do it just in case but everything seems work just fine.

Reply:
done